docs/health/health_design/design.rst

```diff
@@ -1,6 +1,63 @@
 Design
 ======
 
+The previous page showed you how to use pyATS Health Check easily. This page is for those who want to use their own pyATS Health Check. If you are happy with previous page way, please move on :ref:`processor key in YAML<processor_key_in_yaml>`.
+
+Collect and monitor the state of your devices as your testscript is executing with pyATS Health Check. It can collect traceback, core files, etc. pyATS Health Check can be even yaml driven and it is based on :ref:`Blitz<blitz>`. All Blitz functionalities are fully supported in health check.
+
+The health check is driven by a `health_file` which is provided at run time. There is two different mechanism to run the checks:
+
+1. Part of a pre/post processor of a section or testcase.
+2. Part of another process continuously (Coming with 21.7).
+
+To use own pyATS Health Checks, pyATS Health Check leverages existing :ref:`Blitz<blitz>` style YAML format and use same format and capabilities what Blitz has.
+The actions in pyATS Health Check yaml, those actions will be added to pyATS job's each section as pre/post processors. Pre, Post or both processors can be specified in the yaml.
+
+.. note::
+
+     See the detail of :ref:`Blitz<blitz>`
+
+.. code-block:: bash
+
+    # Quick start Guide!
+    # ------------------
+    # 1. Create the health yaml
+    # 2. Add the arguments
```

Comment:
add the arguments ... where? in the yaml file? to the job command line?

Reply:
yes. to pyats command. updated to be clear via 165442b